### ISSUE TITLE
Slug not set on `save(validate: false)`

### ISSUE THREAD

**Issue by dylankarr** [norman/friendly_id#738]
When saving a model without using validations, slugs are not set:

```
u = User.new(some_params)
u.save(validate: false)
u.slug
=> nil
u.save
u.slug
=> 'some-slug'
```

**Comment by PavelJ** [comment]
Well I am not the author, but from looking into the repo it looks that when you try to avoid validation, you have to call set_slug method by hand (send(:set_slug) or so...), see the implementation:
[friendly_id_github_before_validation](https://github.com/norman/friendly_id/blob/master/lib/friendly_id/slugged.rb#L250)

So your code cannot work like that, it was also mentioned on [stackoverflow](http://stackoverflow.com/questions/33834348/creating-a-friendly-id-with-validate-false)

So basically, this is a feature, not a bug but to be honest, it bothers me a bit too.

**Comment by dylankarr** [comment]
@PavelJ thanks
